rusk-wallet/src/bin/command.rs

```diff
@@ -226,7 +231,7 @@ pub(crate) enum Command {
         fn_args: Vec<u8>,
 
         /// Max amount of gas for this transaction
-        #[clap(short = 'l', long, default_value_t = DEFAULT_STAKE_GAS_LIMIT)]
+        #[clap(short = 'l', long, default_value_t = DEFAULT_LIMIT_STAKE)]
```

Comment:
why is stake-related constant used in an object which is not stake-related

Reply:
@miloszm Changed to `DEFAULT_LIMIT`

Reply:
I don't think it is a good idea to take the default gas limit here, since the default limit is the default limit for single transaction. Any contract call (like for example any stake operation) will consume more gas points which is why the default stake limit makes sense here but would need a different name (like I mentioned in my other review comments)